asv_bench/benchmarks/utils.py

```diff
@@ -204,9 +206,8 @@ def gen_str_int_data(nrows: int, ncols: int, rand_low: int, rand_high: int) -> d
         )
     )
     data = gen_int_data(nrows, ncols, rand_low, rand_high).copy()
-    data["gb_col"] = [
-        "str_{}".format(random_state.randint(rand_low, rand_high)) for i in range(nrows)
-    ]
+    # convert values in `col13` column to string type
+    data["col13"] = [f"str_{x}" for x in data["col13"]]
```

Comment:
Now the size of the dataset is the same for all generating functions. Column name is arbitrary.

Reply:
this seems unrelated, why is it done in this PR?

Reply:
To simplify the description of the return value for generation function :)